### ISSUE TITLE
weekly reports?

### ISSUE THREAD

**Issue by matti** [linuxkit/linuxkit#2528]
Was there a decission to stop maintaining weekly reports? they are still linked from the main README.

**Comment by justincormack** [comment]
No, but various things got in the way. I should start doing them again...



On 21 Sep 2017 11:12, "Matti Paksula" <notifications@github.com> wrote:

Was there a decission to stop maintaining weekly reports? they are still
linked from the main README.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/linuxkit/linuxkit/issues/2528>, or mute the thread

**Comment by matti** [comment]
Please do, they are great for people following this project :)

**Comment by arm64b** [comment]
Hi @justincormack  Take use of this chance, can I ask if there is an effective communication way for us such as a technical discussion group that we can discuss with each other, currently the only way for me is to raise a "issue" or just send you guys a personal email(don't want to bother you)?

**Comment by justincormack** [comment]
@arm64b there is a pretty active `#linuxkit` channel in the Docker Communirt Slack which you can sign up for at https://community.docker.com/registrations/groups/4316

We are also planning to set up a mailing list very soon - we meant to do so earlier.

Feel free to email!

We are also thinking of having some regular calls again - we had weekly calls for a while, and the security group calls, but I think it would be helpful to have a general call again. These can be difficult from a timezone point of view though.

What would be the most useful for you?

**Comment by arm64b** [comment]
@justincormack  Slack is useful for me, but if there's some urgent technical issue need to be discussed, we can setup a phone call tentatively. Thanks for the info!
